src/Displaz.jl

```diff
@@ -2,12 +2,17 @@ __precompile__()
 
 module Displaz
 using Compat
-using FixedSizeArrays
 using Colors
 
 export plot3d, plot3d!, clearplot, viewplot
 export KeyEvent, CursorPosition, event_loop
 
+if VERSION > v"0.5-" # Interop with StaticArrays
+    using StaticArrays
+else # Interop with FixedSizeArrays
+    using FixedSizeArrays
```

Comment:
I reckon we should just support `FixedSizeArrays` whenever the user has it installed (should still work in 0.5).

```julia
_use_FixedSizeArrays = Pkg.installed("FixedSizeArrays") != nothing
_use_StaticArrays = VERSION > v"0.5-"  # Or detect installation?

if _use_FixedSizeArrays
    using FixedSizeArrays
end
# ...
if _use_FixedSizeArrays
    function interpret_position{V <: FixedVector{3}}(pos::AbstractVector{V})
         # etc ...
    end
end
```